### PR TITLE
add test case to fix code example in doc

### PR DESCRIPTION
**Pull request by yuan-xy**

### CODE REVIEW

File: jedi/api/__init__.py [hunk 1]
```diff
@@ -759,10 +759,11 @@ def x():
             global_var = 3
 
             def bar(foo):
-                return foo + 1 + global_var
+                return int(foo + 1 + global_var)
 
             def x(foo):
```
Comment (davidhalter) [inline jedi/api/__init__.py:764]: Can you remove this foo in the param, it shouldn't be there.

Comment (yuan-xy) [reply jedi/api/__init__.py:764]: I can't understand why.
before refactor:
```
            global_var = 3

            def x():
                foo = 3.1
                x = int(foo + 1 + global_var)
```
after refactor, as show in test, the result is:
```
            global_var = 3

            def bar(foo):
                return int(foo + 1 + global_var)

            def x(foo):
                foo = 3.1
                x = bar(foo)
```

Comment (davidhalter) [reply jedi/api/__init__.py:764]: No it's not, in your own test you have `def x():` both before and after

**Comment by yuan-xy** [comment]
I see :)

**Pull request merged**